cdk/lambdas/wdiv-s3-trigger/tests/test_github_helper.py

```diff
@@ -40,6 +41,7 @@ def test_raise_issue_without_key(self):
     def test_raise_new_issue_with_key(self):
         repo = "chris48s/does-not-exist"
         key = "f00b42"
+        os.environ["SERVER_ENVIRONMENT"] = "production"
```

Comment:
Oh Gah, yes, I didn't see the file name. Sorry, ignore everything I just said

Reply:
Can you make an issue about this to fix it post-elections? I think it will cause problems in the future, and we want something like [this](https://stackoverflow.com/questions/2059482/temporarily-modify-the-current-processs-environment/34333710#34333710) at some point